### PR TITLE
Add experimental Pyxis support

### PR DESCRIPTION
**Pull request by mike1808**
I just tried to follow the guidelines i received from the previous PR #263 

This is not tested but it might work.

### CODE REVIEW

File: src/classes/devices/acaia/acaia.ts [hunk 1]
```diff
@@ -219,8 +218,8 @@ export class AcaiaScale {
     }
 
     this.worker = new DecoderWorker(this.messageParseCallback.bind(this));
-    this.logger.log("Subscribing to notificatoins", { device_id: this.device_id, weight_uuid: this.weight_uuid, char_uuid: this.char_uuid });
-    ble.startNotification(this.device_id, this.weight_uuid, this.char_uuid, this.handleNotification.bind(this), (err) => {
+    this.logger.log("Subscribing to notificatoins", { device_id: this.device_id, weight_uuid: this.weight_uuid, char_uuid: this.rx_char_uuid });
```
Comment (graphefruit) [inline src/classes/devices/acaia/acaia.ts:221]: Shouldn't be a switch here, for the normal char_uuid and the new rx_char_uuid?

File: src/classes/devices/acaia/acaia.ts [hunk 2]
```diff
@@ -232,7 +231,7 @@ export class AcaiaScale {
 
   public async disconnect() {
     this.connected = false;
-    await promisify(ble.stopNotification)((this.device_id, this.weight_uuid, this.char_uuid));
+    await promisify(ble.stopNotification)((this.device_id, this.weight_uuid, this.tx_char_uuid));
```
Comment (graphefruit) [inline src/classes/devices/acaia/acaia.ts:234]: Same here?

**Comment by mike1808** [comment]
@graphefruit tx stands for transmit and rx for receive. You might be right that it is not clear and too technical. I can rename them to `write_char` and `read_char`

**Comment by graphefruit** [comment]
Its fine, I should have read the whole code aswell ;) 

I'll merge this later on and compile the version

**Comment by graphefruit** [comment]
Thanks for your effort and invest!

**Comment by mike1808** [comment]
Sure! I love giving back to things I use for free :)

**Comment by graphefruit** [comment]
@mike1808 It looks like we still got some issues with the lunar scales.
I opend a ticket, maybe you could comment there in, that I can link you ;)
#266  - I'll add all information theire

**Pull request merged**